apps/web/app/constants.ts

```diff
@@ -77,6 +77,16 @@ export const APP_NAME = process.env.APP_NAME || 'Ever Teams';
 export const APP_SIGNATURE = process.env.APP_SIGNATURE || 'Ever Teams';
 export const APP_LOGO_URL = process.env.APP_LOGO_URL || 'https://app.ever.team/assets/ever-teams.png';
 export const APP_LINK = process.env.APP_LINK || 'https://ever.team/';
+export const APP_SLOGAN_TEXT = process.env.APP_SLOGAN_TEXT;
+
```

Comment:
@samuelmbabhazi @Cedric921 why no default values for all env vars??? What if someone have empty env file???